modules/terrain/README.md

```diff
@@ -0,0 +1,25 @@
+# @loaders.gl/terrain
+
+[loaders.gl](https://loaders.gl/docs) is a collection of framework independent 3D and geospatial parsers and encoders.
+
+This module reconstructs mesh surfaces from height map images, e.g. [Mapzen Terrain Tiles](https://github.com/tilezen/joerd/blob/master/docs/formats.md), which encodes elevation into R,G,B values.
+
+For documentation please visit the [website](https://loaders.gl).
+
+`@loaders.gl/terrain` uses [MARTINI](https://github.com/mapbox/martini) for mesh reconstruction.
+
+ISC License
```

Comment:
We typically don't copy the entire license in every README. Please follow the conventions in other modules (mainly attribution in the main doc/README.md). You can copy LICENSE.md into the root of the module.

Reply:
Is an example of this the 3d-tiles attribution section? I could follow that convention here.